tests/testhttp.nim

```diff
@@ -163,33 +31,11 @@ httpsrv.rpc("noParamsProc") do():
 
 httpsrv.start()
 
-suite "HTTP Server/HTTP Client RPC test suite":
+suite "JSON-RPC test suite":
+  test "Simple RPC call":
+    check waitFor(simpleTest("localhost", Port(8545))) == true
   test "Continuous RPC calls (" & $TestsCount & " messages)":
     check waitFor(continuousTest("localhost", Port(8545))) == TestsCount
-  test "Wrong [Content-Type] test":
-    check waitFor(simpleTest("localhost", Port(8545), 0, 415)) == true
-  test "Bad request header test":
-    check waitFor(simpleTest("localhost", Port(8545), 1, 400)) == true
-  test "Zero [Content-Length] test":
-    check waitFor(simpleTest("localhost", Port(8545), 2, 411)) == true
-  test "PUT/DELETE methods test":
-    check:
-      waitFor(simpleTest("localhost", Port(8545), 3, 405)) == true
-      waitFor(simpleTest("localhost", Port(8545), 4, 405)) == true
-  test "Oversized headers test":
-    check waitFor(headerTest("localhost", Port(8545))) == true
-  test "Oversized request test":
-    check waitFor(bodyTest("localhost", Port(8545))) == true
-  test "HTTP/0.9 and HTTP/1.0 client test":
-    check:
-      waitFor(disconTest("localhost", Port(8545), 5, 200)) == true
-      waitFor(disconTest("localhost", Port(8545), 6, 200)) == true
-  test "[Connection]: close test":
-    check waitFor(disconTest("localhost", Port(8545), 7, 200)) == true
-  test "Omitted params test":
-    check waitFor(simpleTest("localhost", Port(8545), 8, 200)) == true
-  test "Big Content-Length":
-    check waitFor(simpleTest("localhost", Port(8545), 9, 413)) == true
```

Comment:
we're not testing HTTP Server + Client anymore, we just assume that's done in `nim-chronos`